sympy/stats/crv.py

```diff
@@ -169,15 +169,15 @@ def __new__(cls, *args):
     def check(*args):
         pass
 
-    def sample(self):
+    def sample(self, **kwargs):
         """ A random realization from the distribution """
+        return self._sample_python(**kwargs)
         icdf = self._inverse_cdf_expression()
         return icdf(random.uniform(0, 1))
```

Comment:
This code is not reachable due to early return.

Reply:
Yes, I know. I have added my code without deleting the previous code so that we can finalise the prototype.
Also, could you tell me your review about returning the `objs` in `NormalDistribution` under `sympy/stats/crv_types.py`? The current return type is `object`. Is there a better way?